part2/src/components/Button/GradientButton.js

```diff
@@ -7,13 +7,14 @@ const SIZES = {
 };
 
 const GradientButton = styled.button`
-  width: ${({ size }) => SIZES[size] ?? SIZES["medium"]}px;
-  padding: 16px 20px;
+  width: ${({ size }) => SIZES[size ?? "medium"]}px;
```

Comment:
해당 코드는 아래와 같이 수정될 수 있을 것으로 보여집니다.
구조 분해 할당을 사용할때 기본값을 사용할 수 있습니다.
```suggestion
  width: ${({ size = "medium" }) => SIZES[size]}px;
```